Manatee.Trello/Rest/WebApiClient.cs

```diff
@@ -176,11 +177,15 @@ private static HttpContent GetContent(WebApiRestRequest request)
 					formData.Add(content, $"\"{parameter.Key}\"");
 				}
 
-				var byteContent = new ByteArrayContent(request.File);
-				formData.Add(byteContent, "\"file\"", $"\"{request.FileName}\"");
-				TrelloConfiguration.Log.Debug($"\tContent: {formData}");
 
-				return formData;
+                var fileStream = File.Open(request.File, FileMode.Open);
```

Comment:
Here you will need to check whether raw bytes have been uploaded or just a filename; then act appropriately.